web/src/components/TaskPageSwitchComponent.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { TaskPageSwitchComponent } from "@/components/TaskPageSwitchComponent";
+import { generateRoadmap, generateTask } from "@/test/factories";
+import { generateBusiness } from "@businessnjgovnavigator/shared/test";
+import { render, screen } from "@testing-library/react";
+
+describe("TaskPageSwitchComponent", () => {
+  const initialFeatureCigaretteLicense = process.env.FEATURE_CIGARETTE_LICENSE;
+
+  afterEach(() => {
+    process.env.FEATURE_CIGARETTE_LICENSE = initialFeatureCigaretteLicense;
+  });
+
+  describe("FEATURE_CIGARETTE_LICENSE feature flag", () => {
+    const task = generateTask({ id: "cigarette-license" });
+    const roadmap = generateRoadmap({});
+    const business = generateBusiness({});
+
+    const displayContent = {
```

Comment:
[sand] There's a generator you can use for this `generateFormationDbaContent`.

I would reach for that as opposed to hard coding everything here. If `FormationDbaContent` ever changes someone will have to make updates here. It also has the added benefit of reducing the file size.